plio/views.py

```diff
@@ -451,12 +457,31 @@ def metrics(self, request, uuid):
                     "item_type",
                     "question_type",
                     "correct_answer",
+                    "survey",
                 ],
             )
 
             # retain only the responses to items which are questions
             question_df = df[df["item_type"] == "question"].reset_index(drop=True)
-            num_questions = len(questions)
+            # retain only non-survey questions
+            question_df = question_df[~question_df.survey].reset_index(drop=True)
+            num_non_survey_questions = len(
+                Question.objects.filter(item__plio=plio.id, survey=False)
```

Comment:
once you've defined `questions` as I have suggested above, just re-use it:
```suggestion
                questions.filter(survey=False)
```